docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -288,12 +288,7 @@ testcafe all tests/sample-fixture.js -S -s screenshots
 
 ### -q, --quarantine-mode
 
-Enables the quarantine mode for tests that fail.
-In this mode, a failed test is executed several times.
-The test result depends on the outcome (*passed* or *failed*) that occurs most often.
-That is, if the test fails on most attempts, the result is *failed*.
-
-If the test result differs between test runs, the test is marked as unstable.
+Enables the [quarantine mode](programming-interface/runner.md#quarantine-mode) for tests that fail.
```

Comment:
See [Concurrent Test Execution](common-concepts/concurrent-test-execution.md) to learn more about concurrent test execution. - See [Concurrent Test Execution](common-concepts/concurrent-test-execution.md) for more information about concurrent test execution.

If this option is enabled, TestCafe pauses the test at the moment it fails -> If this option is enabled, TestCafe pauses the test **when** it fails

Specifies the time (in milliseconds) within which selectors attempts  -> attempt (selectors - plural)

Specifies the time (in milliseconds) within which TestCafe makes attempts to successfully execute an assertion if a selector property or a client function was passed as an actual value. -> Specifies the time (in milliseconds) TestCafe attempts to successfully execute an assertion if a selector property or a client function was passed as an actual value.

TestCafe **will** correspond these symbols to any number of characters in the URL. Wildcards at the beginning and end of the rules can be omitted (*.mycompany.com and .mycompany.com have the same effect). -> No future tense. 'correspond these symbols' How do you correspond something? Something corresponds to/with.

Enables colors on the command line -> in the command line.